action/config/view.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package config
+
+import (
+	"github.com/go-vela/cli/internal/output"
+
+	"github.com/spf13/afero"
+)
+
+// View inspects the config file based off the provided configuration.
+func (c *Config) View() error {
```

Comment:
This is where we add the new `View()` function.

The purpose of this function is to view a configuration file.